cg/meta/workflow/mutant.py

```diff
@@ -105,6 +105,27 @@ def get_sample_parameters(self, sample_obj: models.Sample) -> MutantSampleConfig
             ),
         )
 
+    def get_internal_NTCs(self, case_id: str) -> List[models.Sample]:
+        """Retrieve a list of lims ids for negative controls created by Clinical Genomics"""
+        case_obj = self.status_db.family(case_id)
+        samples: List[models.Sample] = [link.sample for link in case_obj.links]
+        ticket_id: int = samples[0].ticket_number
+        # 1. ticket_id will lead to samples that all come from the same pool
```

Comment:
One ticket could have several pools, right?

Reply:
The case_id used to start the mutant analysis only point at one ticket. That ticket only point at one pool. In that pool we will find all mutant samples from one plate/case/ticket/order and all internal negative controls relevant for the mutant case.

Reply:
I looked in the Orderform now and there is no column for the customer to name the pool(s) so yes that's right